src/app/dashboard/page.tsx

```
@@ -167,14 +167,14 @@
-                    isPressable
                     className={`bg-content1/50 group relative ${
                       viewType === "list" ? "flex flex-row" : "h-full"
                     }`}
-                    onPress={() =>
-                      router.push(`/dashboard/presentation/${presentation.id}`)
-                    }
                   >
+                    <div 
+                      className="cursor-pointer absolute inset-0 z-10"
+                      onClick={() => router.push(`/dashboard/presentation/${presentation.id}`)}
+                    />
```

ESLint

[error] Avoid non-native interactive elements. If using native HTML is not possible, add an appropriate role and support for tabbing, mouse, keyboard, and touch inputs to an interactive content element.

(jsx-a11y/no-static-element-interactions)